app/controllers/concerns/api/articles_controller.rb

```diff
@@ -103,6 +103,18 @@ def me
         .decorate
     end
 
+    def unpublish
+      @article = Article.find(params[:id])
+
+      authorize @article, :revoke_publication?
+
+      if Articles::Unpublish.call(@article)
+        render json: { message: "OK" }
```

Comment:
Not 100% sure of the best structure for the response. Suggestions for alternatives are welcome here too. Otherwise this is what felt fit okay. 

Otherwise I'm also happy to loop back here later, to look into the API as a whole to identify if all our responses can be refactored into something more consistent, before fully releasing V1 (removing Feature Flag).